### ISSUE TITLE
Emulator is not installed correctly on the M1 MacBook using the setup-dev command

### ISSUE THREAD

**Issue by aaronespasa** [facebookresearch/playtorch#25]
### Version

0.1.2

### Problem Area

CLI

### Steps to Reproduce

This issue only occurs on the M1 MacBook.

To reproduce this issue:

1. If you have the Android Emulator already installed in your computer, uninstall it.
2. Execute the command `npx torchlive-cli setup-dev` to install the dependencies. You will see that the Android Emulator has been installed correctly:
![2FF6C24D-13F8-4C04-B7CF-FAD79D3E7714_4_5005_c](https://user-images.githubusercontent.com/39239895/147966152-d67eb5ec-47f8-4af9-a16d-360e25399c4f.jpeg)

3. Execute the command `emulator -V` and you will see that it won't output the version:
![B2D32A48-883D-4787-98CF-1CAE07DECF59_4_5005_c](https://user-images.githubusercontent.com/39239895/147966243-acfba5a9-b538-45a4-a789-c13b32278b90.jpeg)

### Expected Results

There's an easy fix to this issue. Just going to Android Studio, deleting the Android Emulator (in Appearances & Behavior > System Settings > Android SDK > SDK Tools) and installing it again. Then, it will work correctly.

By doing this, we can execute the command `emulator -V` again and we will see that it works as expected:
![FBEE6C78-2BE4-4FDD-A3A5-6F8AC23F461C_4_5005_c](https://user-images.githubusercontent.com/39239895/147966360-caa4560e-884c-4759-9578-abedca65abaa.jpeg)

However, the Android Emulator should be installed correctly on the M1 MacBooks when executing the `setup-dev` command.

**Comment by raedle** [comment]
Thanks for reporting the issue on M1 devices, @aaronespasa. Is PR #24 related to this issue?

**Comment by aaronespasa** [comment]
@raedle The PR #24 does not solve this issue. The issue that is reported here is related with the installation of the Android Emulator (from Android Studio) and the PR focuses on solving a problem with the creation of the AVD. They are only related in the sense that the AVD will use the Android Emulator.

**Comment by chrisklaiber** [comment]
Closing out this issue since the torchlive-cli has been removed from PlayTorch with version 0.2.2. Thanks for reporting this.

We have released the PlayTorch app and a local development environment is no longer necessary. See https://playtorch.dev/docs/tutorials/get-started/

If a local development environment is needed, we recommend the default approach from React Native or Expo, applied to PlayTorch. See https://playtorch.dev/docs/tutorials/get-started-manually/
